lib/nimble_options.ex

```diff
@@ -590,14 +603,20 @@ defmodule NimbleOptions do
     end
   end
 
-  defp validate_type(:integer, key, value) when not is_integer(value) do
+  defp validate_type(:integer, key, value, redact) when not is_integer(value) do
     error_tuple(
       key,
       value,
-      "invalid value for #{render_key(key)}: expected integer, got: #{inspect(value)}"
+      "invalid value for #{render_key(key)}: expected integer",
+      ", got: #{inspect(value)}",
+      redact
     )
   end
 
+  defp validate_type(_type, _key, value, _redact) do
+    {:ok, value}
+  end
```

Comment:
This is here only for the initial proof-of-concept. This will change as I get further along.